langs.js

```diff
@@ -1880,8 +1858,7 @@ letter:"ငတထပမယရလဝသၵၶၸၺၼၽၾႁ", letteraux:"ဢ
 orth:`Myanmar. &nbsp; <a href="../scripts/myanmar/mymr-shn" target="_blank">Details</a>.`,
 type:"abugida", cs:"no", gpos:"yes", gsub:"yes", cursive:"no", wordsep:"no/zwsp", baseline:"mid", wrap:"?", hyphenation:"?",  direction:"ltr",  justification:"?", 
 vowels:"inh:1 vs:14 vsl:3 base:ဢ/A pre:2 comp:18",  clusters:"mrk",  medials:"sj:3",  finals:"vk:1", 
-linked:"myanmar/mymr-shn", fonts:"mymr-shn",
-fonts:"/pickers/mymr-shn"
+linked:"myanmar/mymr-shn", fonts:"/pickers/mymr-shn"
```

Comment:
The two `fonts` values differed slightly. I'm not sure which one is right, but this one seemed more common throughout.